amr-wind/incflo_advance.cpp

```diff
@@ -399,6 +410,288 @@ void incflo::ApplyPredictor(bool incremental_projection)
     }
 }
 
+// Predictor step for subsequent non-linear iterations within each timestep
+void incflo::ApplyPredictorNonLinear(bool incremental_projection)
+{
+    BL_PROFILE("amr-wind::incflo::ApplyPredictorNonLinear");
```

Comment:
I've mentioned this before but noting it here so I don't forget (it's not you, it's me). I don't like all the duplicate code this triggers with the other ApplyPredictor. It would be good to merge somehow.